libkbfs/mdserver_memory.go

```diff
@@ -465,6 +467,19 @@ func (md *MDServerMemory) GetRange(ctx context.Context, id tlf.ID,
 	}
 }
 
+func (md *MDServerMemory) iTeamMigrationRemoveLock(id tlf.ID) {
```

Comment:
I'm not sure, but it seems likely that these two functions will be called with `md.lock` already held.  But we can fix that when that code is written, I guess.

Reply:
Yeah; good point. I'll fix the in the follow up ticket which will port the migration checking into `MDServerMemory`